apps/func-response-processor/src/index.ts

```
@@ -188,8 +189,14 @@ const serviceBusTrigger = async function (
             timeToZeroBonus: isNaN(parseInt(instanceInfo.timeToZeroBonus, 10))
               ? TIME_TO_ZERO_BONUS
               : parseInt(instanceInfo.timeToZeroBonus, 10),
-            defaultPoints: DEFAULT_POINTS,
-            defaultCorrectPoints: DEFAULT_CORRECT_POINTS,
+            defaultPoints: isNaN(parseInt(instanceInfo.defaultPoints, 10))
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)